### PR TITLE
Tasks now show up in task queue in group view

### PR DESCRIPTION
**Pull request by IshikaJain**
### Summary 

Tasks now show up in task queue in group view. Only the tasks for the user that is logged in show up in the task queue. 

If task is deleted from task queue, task gets deleted from right side and vice-versa. Additionally, when a new task is created for a user, it automatically gets added to the task queue. 

- [x] tasks show up in task queue

### Test Plan 
One task assigned to the user logged in: 
Screen Shot 2020-10-30 at 5 53 11 PM
Screen Shot 2020-10-30 at 5 53 07 PM

No tasks assigned to logged in user: 
Screen Shot 2020-10-30 at 5 52 53 PM
Screen Shot 2020-10-30 at 5 52 41 PM

Many tasks assigned to user: 
Screen Shot 2020-10-30 at 5 52 18 PM

### CODE REVIEW

File: frontend/src/components/GroupView/MiddleBar/index.tsx [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
-);
+const GroupViewMiddleBar = ({
+  groupMemberProfiles,
+  group,
+  changeView,
+  tasks,
+}: Props): ReactElement => {
+  const allInfo: any = localStorage.getItem('firebaseui::rememberedAccounts');
+  const allInfoJSON = JSON.parse(allInfo);
+  const { email } = allInfoJSON[0];
```
Comment (SamChou19815) [inline frontend/src/components/GroupView/MiddleBar/index.tsx:23]: do not use this to get user. We have a better typed version. See

https://github.com/cornell-dti/samwise/blob/72aeb1ddc75401244758d090f9c0c1f8d400807c/frontend/src/firebase/auth-util.ts#L40-L48

File: frontend/src/components/GroupView/MiddleBar/index.tsx [hunk 2]
```diff
@@ -1,25 +1,43 @@
...
+import { Group, SamwiseUserProfile, Task } from 'common/types/store-types';
 
 import GroupViewMiddleBarPeopleList from './GroupViewMiddleBarPeopleList';
 import GroupViewMiddleBarTaskQueue from './GroupViewMiddleBarTaskQueue';
 import styles from './index.module.scss';
 
 type Props = {
   readonly group: Group;
-  readonly groupMemberProfiles: readonly SamwiseUserProfile[];
+  readonly groupMemberProfiles: SamwiseUserProfile[];
```
Comment (SamChou19815) [inline frontend/src/components/GroupView/MiddleBar/index.tsx:10]: why are we removing readonly here

File: frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarTaskQueue.module.scss [hunk 3]
```diff
@@ -1,10 +1,10 @@
 .TaskQueue {
-  height: 60%;
+  height: 60% !important;
 }
 
 .TaskQueue h2 {
-  font-weight: bold;
-  color: #818181;
+  font-weight: bold !important;
+  color: #818181 !important;
```
Comment (SamChou19815) [inline frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarTaskQueue.module.scss:7]: why do we need `!important` here?

**Review by SamChou19815 — COMMENTED** [review]
some suggestions

File: frontend/src/components/GroupView/MiddleBar/index.tsx (hunk 2 above)
Comment (IshikaJain) [reply frontend/src/components/GroupView/MiddleBar/index.tsx:10]: I was getting an error

Comment (SamChou19815) [reply frontend/src/components/GroupView/MiddleBar/index.tsx:10]: I don't see any code in this diff that can cause this. Try to add it back and report what error you get.

**Review by pihucode — APPROVED** [review]
Tested on my end and everything looks good!

**Pull request merged**